### DISCUSSION TITLE
vga display issues

### DISCUSSION THREAD

**Discussion by IAmNoon1995** [hneemann/Digital#1273] [Q&A]
So I have got a 800x600 pixel display working but when I try to get the vga to display at 1280x800 pixels io get the error it says "video mode was not detected (line len = 1681, h Sync len = 136, hNegitive=false, line count = 829,v Sync len = 3, vNegative=false" I do not know what to do I have tried to fix it but it has not worked can i get some help with this.

**Comment by hneemann** [comment]
Each video mode uses different timings for h-sync and v-sync and the sync polarity also differs. This is how the monitor recognizes the video modes. It is not sufficient to simply output a different number of pixels. 

The video modes that are supported and the corresponding timing parameters can be found [here](https://github.com/hneemann/Digital/blob/master/src/main/java/de/neemann/digital/gui/components/graphics/VGA.java).

The mode 1280x800 is supported.

**Comment by IAmNoon1995** [comment]
here is what I have been trying to  use (new VideoMode(60, 83.46, 1280, 64, 136, 200, 800, 1, 3, 24, true, false)); this is the one that I trying to use but I get that error I do not know how to fix I have it set up similar to my display using this display (new VideoMode(60, 40, 800, 40, 128, 88, 600, 1, 4, 23, false, false).

**Reply by IAmNoon1995** [reply]
![IMG_20240329_075058791](https://github.com/hneemann/Digital/assets/165357410/cbbd1e22-551b-4646-9185-36db4612c420)
This is the non working 1280x800 display.
![IMG_20240329_075040480](https://github.com/hneemann/Digital/assets/165357410/d816c12c-5e33-423a-aa7e-66dadb5ade80)
This is the working 800*600 display.
I just did this on my phone I do not know how it will look if it turns out bad sorry.

**Comment by craig-dougan-maersk** [comment]
PR raised: https://github.com/Maersk-Global/infra-k8s-platform/pull/7524

Adds `retina-e8-1`, `retina-e8-2`, `retina-e8-3` nodepools (`Standard_E8ads_v5`, min=2/max=5 per zone, taint `newretina=true:NoSchedule`) alongside the existing E16ads_v5 pools. Once the PR is applied, the existing `retina1/2/3` pools can be cordoned and drained.
